api/python/cellxgene_census/tests/test_acceptance.py

```diff
@@ -70,18 +70,68 @@ def table_iter_is_ok(tbl_iter: Iterator[pa.Table], stop_after: Optional[int] = 2
 
 @pytest.mark.live_corpus
 @pytest.mark.parametrize("organism", ["homo_sapiens", "mus_musculus"])
-def test_incremental_read(organism: str) -> None:
+@pytest.mark.parametrize(
+    ("stop_after", "ctx_config"),
+    [
+        pytest.param(2, None),
+        pytest.param(None, DEFAULT_TILEDB_CONFIGURATION, marks=pytest.mark.expensive),
+    ],
+)
+def test_incremental_read_obs(organism: str, stop_after: Optional[int], ctx_config: Optional[Dict[str, Any]]) -> None:
     """Verify that obs, var and X[raw] can be read incrementally, i.e., in chunks"""
 
-    # open census with a small (default) TileDB buffer size, which reduces
+    # ctx_config=None open census with a small (default) TileDB buffer size, which reduces
```

Comment:
See existing `small_mem_context` in test_hvg.py.  You may just want to promote this to conftest.py and use it.  That has the benefit of explicitly controlling the configuration, whereas the current code is subject to whatever defaults tiledbsoma decides are appropriate.